src/Domain/Condition/VehicleSet.php

```diff
@@ -33,6 +34,11 @@ public function getRestrictedTypes(): array
         return $this->restrictedTypes;
     }
 
+    public function getCritairTypes(): ?array
```

Comment:
Après avoir dbmigrate avec un arrêté existant j'obtiens cette erreur en accédant à sa page de détail

> Warning: foreach() argument must be of type array|object, null given
> src/Application/Regulation/View/VehicleSetView.php:36

Plusieurs options

* Renvoyer un array ici si on a null
* Gérer le null dans le code appelant
* Mettre un DEFAULT [] et passer le champ en NOT NULL dans la DB. "Préférer les NOT NULL"

Je pense la 3e option serait la plus robuste.